plugin/trino-mysql/src/test/java/io/trino/plugin/mysql/TestMySqlCaseInsensitiveMapping.java

```diff
@@ -71,4 +71,10 @@ protected String quoted(String name)
         name = name.replace(identifierQuote, identifierQuote + identifierQuote);
         return identifierQuote + name + identifierQuote;
     }
+
+    @Test
```

Comment:
Seems like all subclasses of BaseCaseInsensitiveMappingTest need this (except SQL Server).

Can you please add it to all of them (Sql Server, Postgres, MySQL)?

Reply:
Also add a TODO as a doc comment for this method linking to the GitHub issue you found.

Reply:
> Seems like all subclasses of BaseCaseInsensitiveMappingTest need this (except SQL Server).
> 
> Can you please add it to all of them (Sql Server, Postgres, MySQL)?

Copy that.

> Also add a TODO as a doc comment for this method linking to the GitHub issue you found.

Copy that.

Reply:
Sql Server already contains a public test method, I'll add dummy test in Postgres.